packages/flutterfire_ui/lib/src/firestore/table_builder.dart

```diff
@@ -819,8 +846,9 @@ class _Source extends DataTableSource {
 
     final doc = _previousSnapshot!.docs[index];
     final data = doc.data();
-
-    return DataRow(
+    final colList = getHeaders().keys.toList();
```

Comment:
Could you rename colList -> columns?

Reply:
Although arguably I'm not sure what the value of this variable is. I prefer having this in the `for` like before – and the `toList` negatively impacts performances

Reply:
I was needed in a previous commit, because I was passing the index instead of the column key name to cellBuilder.
I removed it